src/ui/package_manager.ui

```diff
@@ -6,17 +6,32 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>658</width>
-    <height>784</height>
+    <width>900</width>
+    <height>600</height>
    </rect>
   </property>
   <property name="sizePolicy">
-   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
+   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
-  <layout class="QVBoxLayout" name="verticalLayout_3">
+  <property name="minimumSize">
+   <size>
+    <width>900</width>
+    <height>600</height>
+   </size>
+  </property>
+  <property name="maximumSize">
```

Comment:
Two points of feedback,

(1) I can't resize the window to be any bigger, which feels limiting. Would be nice if I could resize to any size I'd like.

(2) the version is cut off for my by default - 

<img width="1190" height="857" alt="Image" src="https://github.com/user-attachments/assets/2cbd0317-da14-4bd4-8fcb-1b1b5e4632a9" />

Reply:
Adjusted